content/blog/2021/cff-best-practises.md

```diff
@@ -0,0 +1,71 @@
+Title: Best Practises to look out for when submitting talks
+Date: 2021-04-07 17:09
+Author: Vipul Gupta (@vipulgupta2048)
+Category: 2021
+Slug: cfp-best-practises
+Summary: PyCon India 2021 is looking for talks to be presented at the conference. Read on to know when submitting a proposal how you too can make your talk awesome. 
+
+PyCon India is the largest gathering of Pythonistas in India for the Python programming language. The 13th edition of PyCon India will be taking place online from 17th September to 20th September 2021. So, whether you are someone submitting a proposal for talks for the first time or just want to share what you have been up to with Python for a long time. This is YOUR time to shine!
+
+We are looking forward to seeing beginner, intermediate, and advanced proposals on diverse topics from speakers of different expertise, and experience. To a submit a proposal, you don’t need to be a veteran who has spoken at a dozen conferences in the past. For making our conference truly diverse, we are looking for all kinds of people. That’s what our community is comprised of, and that’s what this conference’s schedule should be truly made out of. As you think about your experiences, skills, and knowledge that you would want to bring to the table. We also want you to focus on the proposal that you will be submitting to us. A well-thought-out proposal could mean a chance for it to be selected and presented at PyCon India 2021. Take your time to go through our team's advice on preparing a kickass proposal that turns heads (virtually):
+
+1. **Know the deadlines, follow the Deadlines.** 
+
+Unlike other iterations of the event, this year we are determined to stick to deadlines that have been enforced and try to provide minimal extensions. Hence, please make note of some important dates as mentioned below:
+
+- CFP opens on **5th March 2021**
```

Comment:
Is this really relevant? This date is long gone.

Reply:
Removed